### ISSUE TITLE
`ga_cubes.json` not found

### ISSUE THREAD

**Issue by hespelz** [singer-io/tap-google-analytics#55]
Stitch/GA integration failed with this log
```log
2021-07-19 23:30:23,904Z   main - INFO Running tap-google-analytics version 1.0.3 and target-stitch version 3.2.0
2021-07-19 23:30:25,123Z   main - INFO [smart-services] building smart service producer: topic( com.stitchdata.extractionJobStarted ), use_ssl( true )
2021-07-19 23:30:25,127Z   main - INFO [smart-services] building smart service producer: topic( com.stitchdata.extractionJobFinished ), use_ssl( true )
2021-07-19 23:30:25,128Z   main - INFO [smart-services] building smart service producer: topic( com.stitchdata.streamRecordCount ), use_ssl( true )
2021-07-19 23:30:25,705Z   main - INFO [smart-services] event successfully sent to kafka: com.stitchdata.extractionJobStarted [16] at offset None
2021-07-19 23:30:25,710Z   main - INFO Starting tap to discover schemas: tap-env/bin/tap-google-analytics --config /tmp/tap_discover_config.json --discover
2021-07-19 23:30:26,073Z    tap - INFO Using cached profile_lookup. Will not check Account Summaries API.
2021-07-19 23:30:26,073Z    tap - INFO Discovering standard fields...
2021-07-19 23:30:26,073Z    tap - INFO Refreshing access token.
2021-07-19 23:30:26,271Z    tap - INFO Discovering custom fields...
2021-07-19 23:30:26,820Z    tap - INFO Parsing cube definitions...
2021-07-19 23:30:26,892Z    tap - WARNING Error fetching raw cubes, falling back to local copy. Exception message: Expecting value: line 1 column 1 (char 0)
2021-07-19 23:30:26,893Z    tap - CRITICAL [Errno 2] No such file or directory: '/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/ga_cubes.json'
2021-07-19 23:30:26,894Z    tap - Traceback (most recent call last):
2021-07-19 23:30:26,894Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/client.py", line 278, in get_raw_cubes
2021-07-19 23:30:26,894Z    tap -     cubes_json = cubes_response.json()
2021-07-19 23:30:26,894Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/requests/models.py", line 897, in json
2021-07-19 23:30:26,894Z    tap -     return complexjson.loads(self.text, **kwargs)
2021-07-19 23:30:26,894Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/simplejson/__init__.py", line 516, in loads
2021-07-19 23:30:26,894Z    tap -     return _default_decoder.decode(s)
2021-07-19 23:30:26,894Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/simplejson/decoder.py", line 370, in decode
2021-07-19 23:30:26,894Z    tap -     obj, end = self.raw_decode(s)
2021-07-19 23:30:26,894Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/simplejson/decoder.py", line 400, in raw_decode
2021-07-19 23:30:26,894Z    tap -     return self.scan_once(s, idx=_w(s, idx).end())
2021-07-19 23:30:26,894Z    tap - simplejson.scanner.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2021-07-19 23:30:26,895Z    tap -
2021-07-19 23:30:26,895Z    tap - During handling of the above exception, another exception occurred:
2021-07-19 23:30:26,895Z    tap -
2021-07-19 23:30:26,895Z    tap - Traceback (most recent call last):
2021-07-19 23:30:26,895Z    tap -   File "tap-env/bin/tap-google-analytics", line 33, in <module>
2021-07-19 23:30:26,895Z    tap -     sys.exit(load_entry_point('tap-google-analytics==1.0.3', 'console_scripts', 'tap-google-analytics')())
2021-07-19 23:30:26,895Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/singer/utils.py", line 229, in wrapped
2021-07-19 23:30:26,895Z    tap -     return fnc(*args, **kwargs)
2021-07-19 23:30:26,895Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/__init__.py", line 167, in main
2021-07-19 23:30:26,895Z    tap -     do_discover(client, config)
2021-07-19 23:30:26,895Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/__init__.py", line 135, in do_discover
2021-07-19 23:30:26,895Z    tap -     catalog = discover(client, config, get_view_ids(config))
2021-07-19 23:30:26,895Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/discover.py", line 461, in discover
2021-07-19 23:30:26,895Z    tap -     all_cubes, cubes_lookup = parse_cube_definitions(client)
2021-07-19 23:30:26,895Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/discover.py", line 355, in parse_cube_definitions
2021-07-19 23:30:26,895Z    tap -     raw_cubes = client.get_raw_cubes()
2021-07-19 23:30:26,895Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/client.py", line 282, in get_raw_cubes
2021-07-19 23:30:26,895Z    tap -     with open(local_cubes_path, "r") as f:
2021-07-19 23:30:26,895Z    tap - FileNotFoundError: [Errno 2] No such file or directory: '/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/ga_cubes.json'
2021-07-19 23:30:26,932Z   main - INFO Tap exited abnormally with status 1
2021-07-19 23:30:27,220Z   main - INFO [smart-services] event successfully sent to kafka: com.stitchdata.extractionJobFinished [9] at offset None
2021-07-19 23:30:27,224Z   main - INFO No tunnel subprocess to tear down
2021-07-19 23:30:27,224Z   main - INFO Exit status is: Discovery failed with code 1 and error message: "[Errno 2] No such file or directory: '/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_google_analytics/ga_cubes.json'".

```

when calling this function
```python
def get_raw_cubes(self):
        try:
            cubes_response = self.get("https://ga-dev-tools.appspot.com/ga_cubes.json")
            cubes_response.raise_for_status()
            cubes_json = cubes_response.json()
        except Exception as ex:
            LOGGER.warning("Error fetching raw cubes, falling back to local copy. Exception message: %s", ex)
            local_cubes_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "ga_cubes.json")
            with open(local_cubes_path, "r") as f:
                cubes_json = json.load(f)
        return cubes_json
```

**Comment by WaydeHerman** [comment]
Am experiencing the same issue on our end.

**Comment by dmosorast** [comment]
Missed this issue popping up in the bustle of it all, sorry about that. This should be fixed now by #56 and #57. Closing this issue. Thank you for the report!
